src/legacy/core_plugins/dashboard_embeddable_container/public/embeddable/_index.scss

```diff
@@ -0,0 +1,3 @@
+@import './viewport/index';
+@import './panel/index';
+@import './grid/index';
```

Comment:
So are you still duplicating styles still, or will the ones from the dashboard plugin be removed? It doesn't look like the class names were changed?

Reply:
The ones from the dashboard plugin will be removed (working on the last PR that does that here: https://github.com/elastic/kibana/pull/39126).  I didn't change the names because I think the prefix still makes sense with `dsh`.  But I can change them if you think something else would work better.  like `dec`?

Reply:
That will mean we'll have duplicate styles in master for a bit. If you didn't change any classnames or styles, maybe we leave the files in but just don't import them yet? The styles will be implemented because the old files still exist and we import all CSS on every page.

Reply:
sg, I will comment out the import!

Reply:
Instead of commenting these ones out I commented the top one out